vyxal/elements.py

```diff
@@ -3,9 +3,21 @@
 (that is, functions directly corresponding to Vyxal elements). It's also where
 the python equivalent of command is stored
 """
+
+import itertools
+import math
+import string
 import types
 from typing import Union
 
+import sympy
+
+from vyxal.helpers import *
+from vyxal.LazyList import LazyList
+
+NUMBER_TYPE = "number"
+SCALAR_TYPE = "scalar"
```

Comment:
It feels rather odd to have these as strings, but I suppose it's good enough for now. Might want to use an enum-like thing later on.